index.html

```diff
@@ -87,6 +87,35 @@
           </div>
         </div>
         <h1 class="heading-48">Component Library</h1>
+        <div class="alert alert--info">
+          <p class="alert__message">
+            The component library is a snapshot of the work found in assets-frontend. Currently assets-frontend is being refactored.
+          </p>
+          <h2>Discussing Changes</h2>
```

Comment:
I don't think this is info is needed here. It should live on the homepage and if the hackpad link on each isn't clear or obvious enough then it needs rethinking.